meeting_minutes/220224_SARIF_TC_53.md

```diff
@@ -0,0 +1,166 @@
+# 1. Opening Activities
+
+## 1.1 Opening comments (Co-Chair David)
+
+## 1.2 Introduction of participants/roll call (Co-Chair David)
+
+| First Name | Last Name | Company          | Role(s)                 |
+|:-----------|:----------|:-----------------|:------------------------|
+| Aditya     | Sharad    | Microsoft        | Voting Member           |
+| Chris      | Meyer     | Microsoft        | Voting Member           |
+| David      | Keaton    | Individual       | Chair                   |
+| Eddy       | Nakamura  | Microsoft        | Voting member           |
+| James      | Spoor     | Snyk Ltd.        | Voting Member           |
```

Comment:
James didn't participate in this meeting.

Reply:
Thanks @eddynaka - fixed